JavaFXUserInterface/src/main/resources/css/revamp/assetInfo.css

```diff
@@ -122,4 +122,11 @@
     -fx-font-family: "Segoe UI Semibold";
     -fx-font-size: 20px;
     -fx-padding: 5 15 5 15
-}
\ No newline at end of file
+}
+#assetSaveBtn {
```

Comment:
This save Button doesn't seem to be fitting with the rest of the styling used for the other save buttons on other screens. The color, pref-width, and height as well as other styling are different. See add asset and add asset type for example